packages/rn-tester-e2e/wdio.conf.ios.js

```diff
@@ -0,0 +1,55 @@
+const path = require('path');
+
+exports.config = {
+    runner: 'local',
+    path: '/',
+    
+    specs: [
+        './tests/specs/components/**/*.test.js'
+    ],
+    exclude: [
+    ],
+    
+    maxInstances: 1,
+    
+    capabilities: [{ 
+      platformName: 'iOS',
+      'appium:platformVersion': '17.0',
+      'appium:deviceName': 'iPhone 15',
+      'appium:automationName': 'XCUITest',
+      'appium:app': path.join(process.cwd(), '/apps/rn-tester.app'),
+    }],
+    logLevel: 'info',
+    bail: 0,
+    waitforTimeout: 10000,
+    connectionRetryTimeout: 120000,
+    
+    connectionRetryCount: 3,
+    services: [
+        [
+          'appium',
+          {
+            args: {
+              address: 'localhost',
+              port: 4723
+            },
+          }
+        ]
+      ],
+    
+    reporters: ['spec'],
+    framework: 'mocha',
+    mochaOpts: {
+        retries: 4,
+        ui: 'bdd',
+        timeout: 60000,
+        require: ['@babel/register']
+    },
+
+    afterTest: async function(test, context, { error, result, duration, passed, retries }) {
+        if (!passed) {
+            await driver.takeScreenshot();
```

Comment:
I would suggest building a local file path to export these screenshots too. This way, users who run these tests locally know exactly where the screenshots are located (also add this to the documentation so they know where to look).

For example,

```
    // get current test title and clean it, to use it as file name
    const fileName = encodeURIComponent(test.description.replace(/\s+/g, '-'));

    // build file path
    const filePath = './errorShots/' + fileName + '.png';

    
/await browser.saveScreenshot(filePath);
```

Reply:
To make sure that folder is created beforehand:

```
  beforeSession: (/* config, capabilities, specs */) => {
    fs.mkdirSync('./errorShots', { recursive: true });
  },
```